vaas/vaas/router/models.py

```diff
@@ -120,7 +121,30 @@ def provide_route_configuration():
             Action(action='pipe', name='bypass the cache')
         ],
     )
+class RewriteConfiguration(DictEqual):
+    def __init__(self, http_methods, domains):
+        self.pk = 'configuration'
+        self.http_methods = http_methods
+        self.domains = domains
+
+class HttpMethod(DictEqual):
+    def __init__(self, http_method, name):
+        self.pk = http_method
```

Comment:
It's need this when we got field http_method line bellow ?

Reply:
Removed